lib/srchandler.js

```diff
@@ -66,12 +66,12 @@ const get_pc_to_inst_map = bin_string => {
 // This mess deals with displaying the highlighted compact
 // source code with lines.
 // DONT MESS WITH THIS CODE IF YOU DON'T HAVE TO
-const getCodeStringFromPc = (srcs, contract, pc, trim) => {
+const getCodeStringFromPc = (srcs, contract, pc, trim, isCreate = false) => {
```

Comment:
```suggestion
const getCodeStringFromPC = (srcs, contract, pc, trim, isCreate = false) => {
```

I agree with Go here that acronyms should be uppercased.

Reply:
plz separate PR with cleanup and code refactors, i didn't introduced the function here and don't want to start refactoring it everywhere

Reply:
Sorry, this and the issues above is me being dump while looking at a split diff and not noticing when things were just being moved around. Will switch back to unified diff as apparently I can't read split ones :sweat_smile: